### PR TITLE
Update jobs description and the working

### PR DESCRIPTION
**Pull request by dsamard**
Hey I add the explanation it working

### CODE REVIEW

**Comment by JuliusDanes** [comment]
Thank you I appreciate it very much

**Review by dsamard — COMMENTED** [review]
.

**Review by dsamard — COMMENTED** [review]
Sip

File: README.md [hunk 1]
```diff
@@ -4,3 +4,33 @@ your local todo list
 
 <https://localtodo.now.sh>  
 <https://another-localtodo.netlify.com>
+
```
Comment (dsamard) [inline README.md:7]: *

**Review by dsamard — COMMENTED** [review]
*

**Pull request merged**